src/public.rs

```diff
@@ -112,39 +115,17 @@ impl fmt::Display for PublicKey {
 #[cfg(feature = "std")]
 mod use_std {
     use super::*;
-    use crate::{Error, SecretKey, Signature};
+    use crate::{Error, SecretKey};
 
-    use secp256k1::{
-        recovery::{RecoverableSignature, RecoveryId},
-        {Error as Secp256k1Error, Message, PublicKey as Secp256k1PublicKey, Secp256k1},
-    };
+    use secp256k1::{Error as Secp256k1Error, PublicKey as Secp256k1PublicKey, Secp256k1};
 
     use core::borrow::Borrow;
     use core::str;
 
     const UNCOMPRESSED_PUBLIC_KEY_SIZE: usize = 65;
+    const SECP_UNCOMPRESSED_FLAG: u8 = 4;
```

Comment:
Maybe add a small comment explaining what this is for?

Reply:
https://github.com/FuelLabs/fuel-crypto/pull/2/commits/c00011be14b66910a6340f6281fed29b94c6dd5c